src/UI/Components/Diagram/actions.ts

```diff
@@ -35,7 +35,16 @@ export function showLinkTools(
     target.id as dia.Cell.ID,
   ) as ServiceEntityBlock;
 
-  const check = (cellOne: ServiceEntityBlock, cellTwo: ServiceEntityBlock) => {
+  /**
+   * checks if the connection between cells can be deleted
```

Comment:
deleted, or hidden? method name tells something different.

Reply:
deleted, as the only tool that is there is to delete connection, if attributes are rw the we are editing instance won't show the tool

Reply:
Yeah, okay, but this should be in your docstrings. The method is called shouldHideLinkTools plural. and your description talks about deleting connections.